### ISSUE TITLE
Labels don't work

### ISSUE THREAD

**Issue by kartikluke** [scottdejonge/map-icons#21]
Can't seem to get the labels to show. Searching the DOM I can't find the `label: '<i class="map-icon-parking"></i>'` that was set.

**Comment by makketagg** [comment]
@kartikluke, can you give source code - i had some problems with labels

**Comment by tintoverano** [comment]
hi,

label is not shown for me neither and I get this error too:

````
Uncaught TypeError: undefined is not a function (program):1 
rT                                              (program):1
(anonymous function)                            maps.gstatic.com/maps-api-v3/api/js/20/3/main.js:15 
(anonymous function)                            maps.gstatic.com/maps-api-v3/api/js/20/3/main.js:45
````

I'm using the package in Meteor

thanks,

tinto

**Comment by c9dd** [comment]
Same issue here

**Comment by phonofidelic** [comment]
Had the same problem initially. You need to make sure too include the css styling of the marker-label and marker-icon classes in. This should be made in your main css file:
``` 
.marker-label,
.marker-icon {
    z-index: 99;
    position: absolute;
    display: block;
    margin-top: -50px;
    margin-left: -25px;
    width: 50px;
    height: 50px;
    font-size: 30px !important;
    text-align: center;
    color: #FFFFFF;
    white-space: nowrap;
}
```
and styled to preference 
Hope this helps!

**Comment by scottdejonge** [comment]
Fixed in v3.0.0

**Comment by kontrol** [comment]
@tintoverano, How exactly did you get the package to work in meteor?  I didn't see any packages in atmosphere or did I miss it?
